.circleci/src/workflows/delivery.yml

```diff
@@ -1,17 +1,14 @@
-triggers:
-  - schedule:
-      cron: '0 4 * * 1' # For sanity checks for this syntax: https://crontab.guru/
-      filters:
-        branches:
-          only:
-            - main
+when:
+  and:
+    - << pipeline.parameters.api_triggered >>
```

Comment:
Could be done in a separate issue, will not change this here.

Reply:
Why not?

Reply:
Because I did not introduce this in this PR and it works. And I really have enough stuff on my plate currently.

Reply:
You introduce an unnecessary variable with this PR.

Reply:
thought this was introduced in this PR.

Argument about 'having to much stuff to do', is nothing I can consider when doing a review...

Reply:
It was not introduced here and apart from that it is a very minor thing. You're right, and I like it that you pointed out that it is unnecessary because I asked for more thorough reviews. So sorry for that.

Reply:
Since this has possible values of `api`, `scheduled_pipeline` and `webhook` this would make checking unnecessary complicated.